src/system/group.cr

```diff
@@ -127,7 +127,7 @@ struct System::Group
   # :nodoc:
   def self.check_gid_in_bounds(gid : Int) : Nil
     return if (gid >= 0) && (gid <= GID_MAX)
-    raise OutOfBoundsError.new("gid, #{gid.inspect}, is out of bounds")
+    raise OutOfBoundsError.new("gid #{gid.inspect} is out of bounds")
```

Comment:
`gid` => `Gid` as conventionally the exceptions message starts with an uppercase letter.

Also, here you probably don't want `gid.inspect` but just `gid` as it is a number. In either case, if you still want quotes you'll need to add them manually: `"Gid '#{gid}' is out of bound"`.